### ISSUE TITLE
[Feature Request] Idle Notifications for Boosted Stats Threshold

### ISSUE THREAD

**Issue by zwilkin** [runelite/runelite#9554]
### **Is your feature request related to a problem? Please describe.**
I would like to receive a notification when a boosted stat has dropped to/below a designated threshold. Identical to the _Hispoints Notification_ & _Prayer Notification_ settings.

***

### **Describe the solution you'd like**
_NOTE: I get that adding a threshold to every single stat would be overkill - that's not what I'm proposing..._

My idea is provide the user with the ability to select a desired stat(s), _(and/or possible combinations - i.e. 'Melee' would represent Atk/Str/Def/HP, etc.)_, and assign a threshold to send a notification at.

#### &nbsp;&nbsp;Threshold Types
Number _(static)_ | Percentage _(dynamic)_
-- | --
· A solid number used as a threshold.<br><br>· Mimics current _Hitpoints_ and _Prayer_ <br>&nbsp;&nbsp;thresholds notifications. | · Specified % of total lvl boost used as a threshold.<br>&nbsp;&nbsp;&nbsp;· _% of level (ex: 60/50 = 20%)_<br><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_- OR -_<br><br>· Specified % of boosted amount used as a threshold.<br>&nbsp;&nbsp;&nbsp;· _% of the max boost amount<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(ex: 60/50 = +10 boost, so 20% would invoke at 52/50)_<br>

***

### Describe alternatives you've considered
From what I have found, I have not seen anything other than the _Hitpoints_ & _Prayer_ notification settings. But it would basically be a mimic of that where the user can select a desired stat/stats.

***

### Additional context
Nothing much to add that isn't explained in the details...

**Comment by Alexsuperfly** [comment]
try reading the options in the `boosts information` plugin, its likely got what you want

**Comment by Hydrox6** [comment]
Yeah this is already a feature with the `Boosts Information` Threshold. You'll probably need to enabled `Send notifications when focused` under `RuneLite`

**Comment by zwilkin** [comment]
> Yeah this is already a feature with the `Boosts Information` Threshold. You'll probably need to enabled `Send notifications when focused` under `RuneLite`

Ok but what about, for instance, training combat with 2 drastically different level'ed skills? Like 60 attack, but 90 str. Naturally, the str will have a higher boost which will last longer - and the atk will have a lower boost that runs out faster. Now what if you don't care about the atk boost, just want to re-pot at a designated str boost?

**Comment by Nightfirecat** [comment]
The "Number" threshold type you described in your issue already exists. The percentage threshold does not.

**Comment by zwilkin** [comment]
> The "Number" threshold type you described in your issue already exists. The percentage threshold does not.

Up until this thread, I had not realized that you could set a global threshold number for boosts; however I believe that the percentage threshold as well as selecting custom/specific skills (as mentioned in my previous comment) have good merit.
